docs/manage/parts-and-remotes.md

```diff
@@ -96,4 +96,4 @@ servo = Servo.from_robot(robot=robot, name='my-other-robot-main:my_servo')
 {{% /tab %}}
 {{< /tabs >}}
 
-For an example that controls a motor that is a component of a sub-part, see [the Mock Robot tutorial](https://docs.viam.com/tutorials/configure/build-a-mock-robot/#how-to-control-a-sub-part-using-the-viam-sdk).
+For an example that controls a motor that is a component of a sub-part, see [the Mock Robot tutorial](/tutorials/configure/build-a-mock-robot/#control-a-sub-part-using-the-viam-sdk).
```

Comment:
nice catch!!

Reply:
While this might seem like a _very pendantic_ PR, turns out `htmltest` ignores full URLS, so never caught this and one other malformed anchor target. Fixed now!